### ISSUE TITLE
Unable to locate package nzbdrone

### ISSUE THREAD

**Issue by Barishhh** [MichaIng/DietPi#1502]
#### Required Information:
- DietPi Version | v6.1
- SBC Device (EG: RPi 3)? Odroid C2
- Power supply used (EG: 5v 1A RAVpower)? 5V 3A charger
- SD card used (EG: Sandisk ultra)? Sandisk SD 256GB
- Distro (EG: Jessie) | ```uname -a```? 

#### Additional Information (if applicable):
- Software title? Nzbdrone
- Can this issue be replicated on a fresh installation of DietPi? Affirm.

#### Expected behaviour:
Trying to install Sonarr, but failing.

#### Actual behaviour:
Throws an error at the dietpi-software installation screen while installing Sonarr. 
 
DietPi-Software: G_AGI: nzbdrone
                 │  - exit_code = 100
                 │  - VERSION:v6.1 | HW_MODEL:12 | HW_ARCH:3 | DISTRO:3
                 │
                 │ Log file contents:
                 │ E: Unable to locate package nzbdrone
                 │
                 │ If problems persist, please report this to DietPi for investigation, including a
                 │ screenshot of this error! (https://github.com/Fourdee/DietPi/issues).

#### Steps to reproduce:
Install sonarr yourself using dietpi-software 

#### Did you submit a ```dietpi-bugreport```?
d80ba514-469e-4fff-a964-b8492b0f8757-0

**Comment by Barishhh** [comment]
Tried to add `dpkg --add-architecture armhf `after a reinstallation as suggested in https://github.com/Fourdee/DietPi/issues/1277. Didn't solve the problem and still returns a exit code 100.

**Comment by Fourdee** [comment]
@Barishhh 

Thanks for the report 👍 

~Please can you confirm which software title?~

 >Trying to install Sonarr,

We'll try to replicate.

**Comment by Fourdee** [comment]
Yep, seems armhf is being ignored.

https://wiki.debian.org/Multiarch/HOWTO
```
root@DietPi:~# dpkg --print-architecture
arm64
root@DietPi:~#  dpkg --print-foreign-architectures
armhf

#Should default to armhf
root@DietPi:~# cat /etc/apt/sources.list.d/sonarr.list
deb [arch=armhf] https://apt.sonarr.tv/ master main

root@DietPi:~# apt-get update | grep son
Hit https://apt.sonarr.tv master InRelease
Hit https://apt.sonarr.tv master/main armhf Packages
Get:6 https://apt.sonarr.tv master/main Translation-en_GB [169 B]
Get:8 https://apt.sonarr.tv master/main Translation-en [169 B]
Get:10 https://apt.sonarr.tv master/main Translation-en_GB [169 B]
Get:14 https://apt.sonarr.tv master/main Translation-en [169 B]
Get:17 https://apt.sonarr.tv master/main Translation-en_GB [169 B]
Get:18 https://apt.sonarr.tv master/main Translation-en [169 B]
Get:20 https://apt.sonarr.tv master/main Translation-en_GB [169 B]
Get:23 https://apt.sonarr.tv master/main Translation-en [169 B]
Get:24 https://apt.sonarr.tv master/main Translation-en_GB [169 B]
Ign https://apt.sonarr.tv master/main Translation-en_GB
Get:25 https://apt.sonarr.tv master/main Translation-en [169 B]
Ign https://apt.sonarr.tv master/main Translation-en


root@DietPi:~# apt-get install nzbdrone:armhf
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package nzbdrone

```

---

Another install option:
https://github.com/Sonarr/Sonarr/wiki/Installation#other

**Comment by Barishhh** [comment]
Let me know if I can help in any way!

**Comment by Gabba35** [comment]
Am also having this issue (exit code:100, version 6.4). Thanks for working on it Fourdee.

**Comment by Fourdee** [comment]
Resolved for v6.7.

**Comment by Barishhh** [comment]
👍
